jenkins-slave/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM docker.io/java:openjdk-7-jdk
+MAINTAINER Jess Malvin Chin, jess@teamcodeflux.com
+RUN apt-get update && apt-get install -y curl
+RUN apt-get install -y openssh-server
+RUN mkdir /var/run/sshd
+
+RUN apt-get install -y \
+    make \
+    build-essential \
+    libssl-dev \
+    zlib1g-dev \
+    libbz2-dev \
+    libreadline-dev \
+    libsqlite3-dev \
+    wget \
+    curl \
+    llvm
```

Comment:
Please run `rm -rf /var/lib/apt/lists/*` to remove any cached artifacts. This should make the image smaller.